### DISCUSSION TITLE
文件排序混乱了，

### DISCUSSION THREAD

**Discussion by 99xx9** [midoks/mdserver-web#540] [General]
文件排序升级以后，无法按文件名称排序了，还有无法显示每页100个 1000个 2000个   这样的文件显示无效了

**Comment by midoks** [comment]
文件排序升级以后，无法按文件名称排序了 ? 原来也没有呀。
每页100个 1000个 2000个 这样的文件显示无效了？ 这个修复好了。

**Comment by 99xx9** [comment]
如何更新到修复好的版本啊

**Reply by midoks** [reply]
mw update_dev && 清空缓存并强制加载
